### DISCUSSION TITLE
Politiker Daten und Übersetzung

### DISCUSSION THREAD

**Discussion by LeonMunz** [irgroup/dis25-2021#15] [Q&A] [answered]
Hallo Zusammen,

könnten wir die zusätzlichen Daten zu den ESuPol-Datensätzen zur Verfügung gestellt bekommen?
Ich denke, das betrifft mehrere Gruppen, in unserem Fall geht es speziell um die Politikerinformationen aus dem Europa-Datensatz, die auch in der letzten Vorlesung angesprochen wurden.

Falls möglich, würden wir auch gerne die Tipps&Tricks für die Übersetzung und die Bestimmung der Sprache in Anspruch nehmen.

Vielen Dank und schönen Sonntag.

**Comment by HaakFab** [comment, marked as answer]
Hallo, 
die verfügbaren demografischen Informationen zu den Politikern aus dem BTW Dataset habe ich soeben ins Repository geladen, die Daten zu den europäischen Politikern kommen nach!

Zu den Übersetzungen:
Backtranslation: In einem Projekt haben wir es so gehandhabt, dass wir für alle "unique" Suggestions via Google Translate die Sprache bestimmt und ins Englische übersetzt haben. Anschließend haben wir dann von von der Englischen Übersetzung zurück in die automatisch erkannte Sprache übersetzt. Sind der hier resultierende Term und der Ausgangsterm identisch, kann davon ausgegangen werden, dass die Sprache korrekt erkannt wurde und die englische Übersetzung zumindest ansatzweise richtig ist. Aber vorsicht: An manchen Stellen sollten Sie Stemming und/oder Lemmatisierung durchführen, um Probleme mit Translation Bias wie "Ärztin --[DE]--> doctor ----> Arzt" zu vermeiden. Dies muss natürlich abhängig von Ihren Forschungsfragen und ihrem methodischen Ansatz begründet entschieden werden. Ansonsten noch weitere Gedanken/Tipps zu Übersetzungen:
- Die offizielle Google Translate API ist nicht kostenlos, aber in Google Sheets kann man mit den Funktionen DETECTLANGUAGE(Text_oder_Bereich) und GOOGLETRANSLATE(Text; [Ausgangssprache]; [Zielsprache]) auch kostenlos, wenn auch etwas umständlich kostenlos auf die Funktionen zugreifen
- Manuelle Überprüfung der erkannten Sprachen oder Korrektur von Übersetzungen kann trotzt der "Backtranslation Kontrolle" sinnvoll sein. Aufgrund der großen Anzahl von Termen macht es aber vermutlich Sinn, hier begründete Priorisierungen zu setzen anstatt zigtausende Übersetzungen zu kontrollieren.

PS.: Sie finden jetzt auch Term Paper Guidelines zum Verfassen und zur Bewertung der Papers im Repository!

Frohes Schaffen, ich freue mich auf Ihre Ergebnisse!
